feedback-api/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.19
```

Comment:
Can we use one common Dockerfile https://github.com/LeastAuthority/feedback-api/blob/dev/Dockerfile? Changes could be combined.